### DISCUSSION TITLE
小丑牌 Balatro|官方中文|本体+1.0.6升补|NSZ|原版

### DISCUSSION THREAD

**Discussion by match-fire** [share121/inter-knot#1441] [其他]
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/2379780/capsule_616x353.jpg?t=1708440464)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/2379780/ss_075cb45cbfa97d8139fb11f21667d6e35f908640.600x338.jpg?t=1708440464)
Balatro是一款以扑克为灵感的魔幻建牌游戏，你将全情投入于打造强大的增效并取得巨大胜利。

无尽可能 易学难精

将有效的扑克手牌与独特的小丑牌相结合，打造不同的增效和策略。赢得足够的筹码，打破盲注的奸计，同时在游戏过程中揭示隐藏的奖励牌手和牌组。为了到达终极盲注、击败最终赌注并稳操胜券，你需要尽一切可能获取优势。

一场扑克风格的终极博弈之旅。
无尽可能 – 每一次抽牌、弃牌和小丑牌的出现都可能彻底改变你的游戏进程。
深度战役模式，包括8种不同难度，还有挑战、每日通关和自定义游戏模式。

 

打破游戏界限 :

灵活运用一系列强大的工具：不同的牌组、小丑牌、塔罗牌、星球牌、幽灵牌和代金券。利用这些多面手工具，点燃连击狂潮，为你的扑克手牌制造出让人兴奋、肾上腺素飙升的增效。

150张小丑牌，每张都有独特的能力
15副不同牌组，每轮游戏可供选择的修改器各不相同
每张小丑牌有5种潜在变体，增加额外的能力
22种塔罗牌变体，可增强你的普通扑克牌
11张星球牌，助你提升可能获得的扑克手牌
32种代金券，可为当前回合提供永久性增益

 

找到自己的节奏

沉浸在Balatro别具特色的迷幻世界中。让合成波音乐声音的冲击萦绕耳际 – 放松心情，逃离日常琐事，准备进入终极的回合制游戏状态。

通过每轮游戏不断完善你的收藏，发现新的卡牌和秘密。
利用详尽的屏幕统计来研究你的最佳牌组、最常使用的小丑牌等等！
沉浸在CRT风格的像素艺术中，享受细致的、手工雕刻的画面。

 

链接: https://pan.baidu.com/s/1xATynxba9Q1ls7n2yV9SQw

提取码: q3dx

解压密码:laoquzhang.com

**Comment by lllliufeng** [comment]
老司机

**Comment by Thina-chng** [comment]
刚买完正版就看到这个......
